app_005/StartViewController.swift

```diff
@@ -79,7 +86,7 @@ class StartViewController: UIViewController {
         super.viewDidLoad()
         if (PFUser.current() != nil){
             print("User signed up and logged in")
-            self.goToTabs()
```

Comment:
Need to uncomment this or else users will go to the onboarding screens every time